dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/program/TrackedEntityDataValueChangeLogDeletionHandler.java

```diff
@@ -44,14 +44,9 @@ public class TrackedEntityDataValueChangeLogDeletionHandler extends DeletionHand
   @Override
   protected void register() {
     whenDeleting(DataElement.class, this::deleteDataElement);
-    whenDeleting(Event.class, this::deleteEvent);
   }
 
   private void deleteDataElement(DataElement dataElement) {
     trackedEntityDataValueAuditService.deleteTrackedEntityDataValueChangeLog(dataElement);
   }
-
-  private void deleteEvent(Event event) {
-    trackedEntityDataValueAuditService.deleteTrackedEntityDataValueChangeLog(event);
```

Comment:
when deleting an event we were deleting its change logs too, but I don't see us doing it in the importer.
https://github.com/dhis2/dhis2-core/blob/master/dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/bundle/persister/DefaultTrackerObjectsDeletionService.java#L112

am I missing something or is it intentional?

Reply:
nevermind, I just saw you add in this PR the change log deletion while deleting the event